send/ftps_generic

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+import os
+import sys
+from ftplib import FTP_TLS
+
+service_name = "ftps_generic"
+port = 21
+
+if __name__ == "__main__":
+	# Parse command line arguments
+	if len(sys.argv) != 4 and len(sys.argv) != 3:
+		print("Error: Expected number of arguments is 2 or 3 (FACILITY_NAME, DESTINATION and optional REMOTE_LOCATION)", file=sys.stderr)
+		exit(1)
+	
+	facility_name = sys.argv[1]
+	destination = sys.argv[2]
+	if len(sys.argv) == 3:
+		# if there is no remote location, use default "/" location
+		remote_folder_path = '/'
+	else:
+		remote_folder_path = sys.argv[3]
```

Comment:
```suggestion
	destination = sys.argv[2]
	# expected destination type url, example: 'ftp://example.org:2121/path-to/folder/'
	parsed_destination = urllib.parse.urlparse(destination, scheme='ftp')
	if parsed_destination['scheme'] not in ['ftp', 'ftps', 'ftpes']:
	    # TODO: print("only FTPS is supported by this service"); exit();
	destination = parsed_destination['hostname']
	port = parsed_destination['port'] or port
	remote_folder_path = parsed_destination['path'].rstrip('/') + '/'
```

Reply:
it should throw a [ValueError](https://docs.python.org/3/library/exceptions.html#ValueError) when an invalid URL is specified, which you can catch or leave, depending on the convention used by other services